torch/_inductor/graph.py

```diff
@@ -673,7 +673,7 @@ def compile_to_module(self):
         for name, value in self.constants.items():
             setattr(mod, name, value)
 
-        log.debug("Output code written to: %s", mod.__file__)
+        output_code_log.debug("Output code written to: %s", mod.__file__)
```

Comment:
Can we have it so it's still possible to get the filename ouput without dumping the code. It's pretty annoying to have to scroll past pages of console output to get to the filename.

Reply:
As it's going to require a bit more effort to do that, let's do it in a different PR. Leaving it as `info` now so that it's already future-proof.

Reply:
Actually, this can be done setting `TORCH_LOGS="inductor"` in this PR. In master, it can be done setting `+inductor`. While writing this PR I've realised that the logging system has some issues as well, see https://github.com/pytorch/pytorch/issues/99144